### DISCUSSION TITLE
Using new serialization module

### DISCUSSION THREAD

**Discussion by sdelamo** [micronaut-projects/micronaut-guides#717] [Ideas]
I would like for us to use the new serialization module in every guide. It will involve adding the serialization feature to every guide and start annotating POJOs with `@Serdeable` when necessary. And explaning the user what does the `Serdeable` annotation does.

@alvarosanchez @burtbeckwith @graemerocher @shingjo thoughts?

**Comment by graemerocher** [comment]
sounds good, but maybe it should be GA before we merge it into all guides

**Reply by sdelamo** [reply]
Agree. 

However, I think we should test it to see if we find anything before GA.
